src/K2/jubin/Solution.java

```diff
@@ -0,0 +1,41 @@
+package K2.jubin;
+
+import java.util.*;
+
+class Solution {
+    public int solution(int[][] board, int[] moves) {
+        int answer = 0;
+        int board_len = board[0].length;
+        Map<Integer, Stack<Integer>> crain_map = new HashMap<>();
```

Comment:
```suggestion
        Map<Integer, Stack<Integer>> craneMap = new HashMap<>();
```
단어 스펠링을 조금 고쳐서 이렇게 쓰면 좋을것같습니다~

Reply:
저도 @JungHyunLyoo 의견에 전적으로 동의합니다